src/network/clientpackethandler.cpp

```diff
@@ -212,29 +212,24 @@ void Client::handleCommand_AccessDenied(NetworkPacket* pkt)
 	u8 denyCode;
 	*pkt >> denyCode;
 
-	if (denyCode == SERVER_ACCESSDENIED_SHUTDOWN ||
-			denyCode == SERVER_ACCESSDENIED_CRASH) {
-		*pkt >> m_access_denied_reason;
-		if (m_access_denied_reason.empty())
+	if (pkt->getRemainingBytes() <= 0)
+		return;
```

Comment:
this looks incorrect. `denyCode`is not handled if the string is not provided

Reply:
Fixed in acc9bf358